ui/test/run_differential_gate.sh

```
@@ -0,0 +1,92 @@
+# ui/test/differential_corpus.hpp. A Mutation added there without moving this
+# number would be a case the sweep silently stops covering — which is how a
+# mutation suite quietly becomes decoration.
+MUTATIONS="${MUTATIONS:-9}"
+FLAGS="-std=c++20 -O1 -Wall -Wextra -Werror"
+INC="-I ui/include -I ui/test"
+
+WORK="$(mktemp -d "${TMPDIR:-/tmp}/forge_differential.XXXXXX")"
+cleanup() {
+  rm -rf "$WORK"
+  [ -d "$WORK" ] && echo "[differential] WARNING: kept $WORK — rm -rf did not remove it"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)